static/js/lib/queries/digitalCredentials.js

```diff
@@ -0,0 +1,25 @@
+import { getCookie } from "../api"
+import type { DigitalCredential } from "../../flow/courseTypes"
+
+export default {
+  downloadDigitalCredentials: (uuid: string, isCourse: boolean) => ({
+    queryKey: "digitalCredentialDownload",
+    url:      `/api/v1/${
+      isCourse ? "course_run_certificates" : "program_certificates"
+    }/${uuid}/request-digital-credential/`,
+    transform: (json: DigitalCredential) => ({
+      digitalCredential: json
```

Comment:
Suggest: similar to the above - rename to `digitalCredentialRequest`

Reply:
Updated.